docs/extensions/s3.md

```diff
@@ -0,0 +1,47 @@
+# S3 as Storage backend
+
+## I chose S3 when initializing the repository!
+
+If you are reading the section you are lucky! you will only have to perform a few configuration changes. If on the contrary you didn't choose S3 as storage when initializing the repository file system, [navigate below](#i-didnt-choose-s3-when-initializing) to follow more in depth changes!
+
+### Create a bucket
+
+InvenioRDM uses a [minio](https://min.io/download) container for S3 alike storage. To create a bucket in minio you can:
+
+a. Install the [mc tool](https://docs.min.io/docs/minio-client-quickstart-guide.html) and run the following command, where `defaul-bucket` is the name of your bucket.
+
+``` console
+$ mc mb default-bucket
+```
+
+b. Use the web UI (https://localhost[:9000]/minio). In the bottom right you will see a `+` sign, click there and add a new bucket.
+
+### Create a new location
+
+Now you need to tell InvenioRDM where is that bucket, for that create a new location. If you are running a local (development) instance, run:
+
+``` console
+$ cd path/to/your/instance
+$ pipenv run invenio files location s3-default s3://default-bucket --default
```

Comment:
It will be `files location create` by the time the docs come out, right?

Reply:
hopefully :)